MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include atool/templates *.html
```

Comment:
Apparently that doesn't include things if you do `make sdist`. Doesn't matter here because we don't do that, but this seemed better for that reason. See e.g. http://stackoverflow.com/a/1857436 and http://stackoverflow.com/a/6714408